Source/HyprMXAdapter.swift

```diff
@@ -65,10 +66,15 @@ final class HyprMXAdapter: PartnerAdapter {
             UserDefaults.standard.set(gameID, forKey: GAMEID_STORAGE_KEY)
         }
 
+        // UserDefaults.standard.bool defaults to false if key is not present
+        let savedAgeRestrictedUserSetting = UserDefaults.standard.bool(forKey: AGE_RESTRICTED_USER_KEY)
         // HyprMX.initialize() uses WKWebView, which must only be used on the main thread
         DispatchQueue.main.async { [self] in
+            // consentStatus will be updated by setGDPR & setCCPA after init
             HyprMX.initialize(withDistributorId: distributorId,
                               userId: gameID,
+                              consentStatus: CONSENT_STATUS_UNKNOWN,
```

Comment:
If there is any public documentation page on HyprMX website for this and/or the `com.chartboost.adapter.hyprmx.ageRestrictedUser` thing it would be great to paste a link somewhere for future reference.

Reply:
[done](https://github.com/ChartBoost/chartboost-mediation-ios-adapter-hyprmx/pull/11/commits/8cbe09d0731b9ae2bede25a4cbe69510820435ac)